server/src/paypal-client.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+/**
+ *
+ * PayPal Node JS SDK dependency
+ */
+const checkoutNodeJssdk = require('@paypal/checkout-server-sdk');
+
+/**
+ *
+ * Returns PayPal HTTP client instance with environment that has access
+ * credentials context. Use this instance to invoke PayPal APIs, provided the
+ * credentials have access.
+ */
+function client() {
+  return new checkoutNodeJssdk.core.PayPalHttpClient(environment());
+}
+
+/**
+ *
+ * Set up and return PayPal JavaScript SDK environment with PayPal access credentials.
+ * This sample uses SandboxEnvironment. In production, use LiveEnvironment.
+ *
+ */
+function environment() {
+  let clientId = process.env.PAYPAL_CLIENT_ID;
+  let clientSecret = process.env.PAYPAL_CLIENT_SECRET;
+
+  return new checkoutNodeJssdk.core.LiveEnvironment(
+    clientId, clientSecret
+  );
+}
+
+async function prettyPrint(jsonData, pre=""){
+  let pretty = "";
+  function capitalize(string) {
+    return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
+  }
+  for (let key in jsonData){
+    if (jsonData.hasOwnProperty(key)){
+      if (isNaN(key))
+        pretty += pre + capitalize(key) + ": ";
```

Comment:
Same with 'pre'. Can you come up with a better name?

Reply:
> Same with 'pre'. Can you come up with a better name?

Like I replied above, I'm also not sure what it is in 100%. I decided it's better to not touch it.